apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx

```diff
@@ -63,6 +64,18 @@ export default class AdminCohortView extends React.Component {
     }
   }
 
+  sanitizeStringForCsv = str => {
+    let res = str;
+    if (str) {
+      // Convert line breaker to dot, multiple spaces to single space
+      res = res.replace(/\n|\r\n|\r+/gm, '.');
```

Comment:
Let's put a space after the period to make it easier to read

Reply:
fixed